solidity/test/utils/InternalCallsVerifier.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.4 <0.9.0;
+
+contract InternalCallsVerifier {
+  function calledInternal(bytes memory _encodedCall) external view {}
+}
```

Comment:
what if we create an abstract contract and call `InternalCallsVerifier(this).calledInternal()`?

Reply:
doesn't work on constructor, yikes